templates/basic-security.groovy

```diff
@@ -0,0 +1,16 @@
+#!groovy
+
+import jenkins.model.*
+import hudson.security.*
+
+def instance = Jenkins.getInstance()
+
+println "--> creating local user 'admin'"
+
+def hudsonRealm = new HudsonPrivateSecurityRealm(false)
+hudsonRealm.createAccount('{{ jenkins_admin.username }}','{{ jenkins_admin.password }}')
+instance.setSecurityRealm(hudsonRealm)
```

Comment:
Aren't the scripts under `/var/lib/jenkins/init.groovy.d/` run on *every* startup and not just the first? If so, I think this script will remove any users that may have been added since the last role application. Also, if the installation is configured to use [matrix-based security](https://wiki.jenkins-ci.org/display/JENKINS/Matrix-based+security), that configuration will be lost when the authorization strategy is replaced.